packages/codemodel.go/src/gocodemodel.ts

```diff
@@ -1502,6 +1502,7 @@ export class ResumeTokenParameter implements ResumeTokenParameter {
     this.paramType = 'optional';
     this.byValue = true;
     this.location = 'method';
+    this.description = 'Resumes the LRO from the provided token.';
```

Comment:
Should we spell out the acronym as `long-running operation (LRO)`? I'm imagining many readers will figure out what the token is for by its name but not understand the acronym

Reply:
Good point, I've made this change.